adapter-excel/src/main/java/ch/prevo/open/node/adapter/excel/ErrorWriter.java

```diff
@@ -0,0 +1,181 @@
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".name")) {
+                            final Cell cell = row.getCell(NAME_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".additionalName")) {
+                            final Cell cell = row.getCell(ADDITIONAL_NAME_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".street")) {
+                            final Cell cell = row.getCell(STREET_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".postalCode")) {
+                            final Cell cell = row.getCell(POSTAL_CODE_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".city")) {
+                            final Cell cell = row.getCell(CITY_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        } else if (propertyPath.endsWith(".iban")) {
+                            final Cell cell = row.getCell(IBAN_COLUMN_INDEX);
+                            cell.setCellStyle(errorStyle);
+                            markError(drawing, cell, violation);
+                        }
+                    }
+                }
+        );
+    }
+
+    private void markError(Drawing<?> drawing, Cell cell, ConstraintViolation<? extends AbstractEmployment> violation) {
+        Comment comment = cell.getCellComment();
+        if (comment == null) {
+            final ClientAnchor anchor = factory.createClientAnchor();
+            comment = drawing.createCellComment(anchor);
+            cell.setCellComment(comment);
+        }
+        final String content = comment.getString() != null ? comment.getString().getString() + "\n" : "";
+        final RichTextString str = factory.createRichTextString(content + violation.getMessage());
+        comment.setString(str);
+    }
+
+    private Optional<Row> findRow(Sheet sheet, AbstractEmployment rootBean) {
+        for (final Iterator it = sheet.rowIterator(); it.hasNext(); ) {
+            final Row row = (Row) it.next();
+            if (Objects.equals(row.getCell(OASI_COLUMN_INDEX).getStringCellValue(), rootBean.getEmploymentInfo().getOasiNumber())) {
+                return Optional.of(row);
+            }
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public void close() throws IOException {
```

Comment:
Maybe we should autoSize the columns at some point as well? Makes it better readable.

Reply:
Good idea